src/oatpp/network/tcp/Connection.cpp

```diff
@@ -173,7 +185,19 @@ v_io_size Connection::read(void *buff, v_buff_size count, async::Action& action)
 
   if(result < 0) {
     auto e = errno;
-    if(e == EAGAIN || e == EWOULDBLOCK){
+
```

Comment:
Let's suppress warning for the whole function - 
just to make code more readable.

```cpp
#ifdef __GNUC__
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wlogical-op"
#endif

function {
 ...
}

#ifdef __GNUC__
#pragma GCC diagnostic pop
#endif

```

Reply:
done